sound/soc/intel/boards/skl_hda_dsp_common.c

```diff
@@ -132,6 +137,14 @@ struct snd_soc_dai_link skl_hda_be_dai_links[HDA_DSP_MAX_BE_DAI_LINKS] = {
 		.no_pcm = 1,
 		SND_SOC_DAILINK_REG(dmic16k, dmic_codec, platform),
 	},
+	{
+		.name = "SSP%d-BT",
```

Comment:
It is a novel way, but I would rather not set this string as initial name.

Reply:
fixed